CHANGES.md

```diff
@@ -40,18 +40,24 @@ To be released.
     [[#613], [#727]]
  -  `Block<T>` class became not to implement `ISerializable`.  [[#751]]
  -  `Transaction<T>` class became not to implement `ISerializable`.  [[#751]]
- -  `Block<T>.ToBencodex()` became to return `Bencodex.Types.Dictionary`.  [[#751]]
- -  `Transaction<T>.ToBencodex()` became to return `Bencodex.Types.Dictionary`.  [[#751]]
+ -  `Block<T>.ToBencodex()` became to return `Bencodex.Types.Dictionary`.
+    [[#751]]
+ -  `Transaction<T>.ToBencodex()` became to return `Bencodex.Types.Dictionary`.
+    [[#751]]
  -  Removed `Block<T>.FromBencodex(byte[])` method.  [[#751]]
  -  Removed `Transaction<T>.FromBencodex(byte[])` method.  [[#751]]
  -  `Block<T>.ToBencodex()` became to take no arguments.  [[#749], [#757]]
+ -  Removed `Swarm<T>.BroadcastBlocks(IEnumerable<Block<T>>)` method.
+    Instead, added `Swarm<T>.BroadcastBlock(Block<T>)` method.  [[#764]]
```

Comment:
IMHO it's better to write a changelog about the added `Swarm<T>.BroadcastBlock(Block<T>)` method in the *Added APIs* section.